server/src/utils/dateUtils.ts

```diff
@@ -0,0 +1,7 @@
+export const addDays = function addDays(date, days) {
```

Comment:
let's use either 

`export const addDays = (date, days) => `

or 

`export Fn addDays {...}`

I prefer the first option as it's used everywhere in the app